fiftyone/utils/eval/regression.py

```diff
@@ -293,12 +309,27 @@ def evaluate_samples(
             _confs = confs
             _ids = ids
 
+        # Metric operators.
```

Comment:
Tested with `AbsoluteError` and `Mean` operators: https://github.com/manushreegangwar/custom_metrics/blob/main/eval_metrics/__init__.py

Reply:
Sample showing absolute error 'eval' and absolute error 'eval_absolute_error' computed using metric operator: 
```
<Sample: {
    'id': '674e42f0567b7e1228157c7a',
    'media_type': 'image',
    'filepath': '/home/manushree/fiftyone/quickstart/data/000880.jpg',
    'tags': ['validation'],
    'metadata': None,
    'created_at': datetime.datetime(2024, 12, 9, 16, 33, 19, 703000),
    'last_modified_at': datetime.datetime(2024, 12, 9, 16, 33, 20, 27000),
    'ground_truth': <Regression: {
        'id': '67571bcfaf69d19398e8e084',
        'tags': [],
        'value': 0.11694350486842164,
        'confidence': None,
    }>,
    'predictions': <Regression: {
        'id': '67571bcfaf69d19398e8e085',
        'tags': [],
        'value': 0.6432772871473967,
        'confidence': 0.8470792054938999,
    }>,
    'weather': 'rainy',
    'eval': 0.5263337822789751,
    'eval_absolute_error': 0.5263337822789751,
}>
```


```
mean absolute error       5.07
mean eval absolute error  5.07
```